### DISCUSSION TITLE
Произвольный откат кода

### DISCUSSION THREAD

**Discussion by amd64fox** [Chimildic/goofy#41] [Помощь с ошибкой] [answered]
Возможно, не у меня одного такая проблема, уже не первый раз google откатывает мой main на пару изменений назад и откатывает не весь код, а частями, рандомно. Далее тебе приходиться перебирать весь код, где-то дописывать, где-то исправлять, не очень прикольное занятие, даже с учетом того, что ты сам это все писал и знаешь, как все восстановить, просто банально на это пустое занятие уходит не мало время. 
Из того что приходит в голову это делать резервную копию в конце дня, например в 00:00 и складывать в папку на гугл диске, т.к по моим наблюдениям если гугл такую шляпу делает, то это происходит ночью.

**Comment by Chimildic** [comment]
Странное описание проблемы. У меня такого никогда не было. Вы точно сохраняете файл до выхода со страницы?

**Reply by amd64fox** [reply]
Да, конечно сохраняю. 
Пишу в основном узнать мб у кого то было или есть тоже самое. У меня такое было раза 3 и все 3 раза я уверен, что сохранил код. Пока делаю резерв сам в ручную.

**Reply by vlalog** [reply]
Вы, случаем, не обновляетесь с телефона?

**Reply by amd64fox** [reply]
Нет, всегда с ПК. На телефоне могу только запустить функцию и то редко.

**Comment by Chimildic** [comment, marked as answer]
Проблема еще актуальна или можно закрыть топик?

**Reply by amd64fox** [reply]
Больше проблема не повторялась и слава богу.
